django_project/certification/models.py

```diff
@@ -0,0 +1,302 @@
+
+import os
+import logging
+import string
+import re
+from django.core.urlresolvers import reverse
+# from __future__ import unicode_literals
+from django.db import models
+from django.utils.text import slugify
+from django.conf.global_settings import MEDIA_ROOT
+from django.utils.translation import ugettext_lazy as _
+from core.settings.contrib import STOP_WORDS
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from unidecode import unidecode
+
+
+logger = logging.getLogger(__name__)
+
+class ApprovedTrainingCenterManager(models.Manager):
+    """ Custom training centre manager that shows only approved conveners """
+
+    def get_queryset(self):
+        """ Query set generator """
+        return super(
+            ApprovedTrainingCenterManager, self).get_queryset().filter(approved=True)
```

Comment:
this will make error because what models manager is making field that call query
super(ApprovedTrainingCenterManager, self).get_queryset().filter(approved=True)

but TrainingCenter doesn't has approved field